packages/common/utils/src/Numbers.ts

```diff
@@ -34,14 +34,22 @@ export const floorToUnit = createNumberFormatterBy(Math.floor);
 
 export const roundToUnit = createNumberFormatterBy(Math.round);
 
-export function formatToKoreanNumber(value: number, options: { floorUnit?: number; formatAllDigits?: boolean } = {}) {
-  const flooredVal = floorToUnit(value, options.floorUnit || 1);
+export function formatToKoreanNumber(
+  value: number,
+  options: { floorUnit?: number; ceilUnit?: number; formatAllDigits?: boolean } = {},
+) {
+  let val = 0;
+  if (options.floorUnit > 0) {
+    val = floorToUnit(value, options.floorUnit || 1);
+  } else if (options.ceilUnit > 0) {
+    val = ceilToUnit(value, options.ceilUnit || 1);
+  }
 
-  if (flooredVal === 0) {
-    return '0';
+  if (val === 0) {
+    return "0";
```

Comment:
"slash" uses "singlequote".

You can refer this prettierrc file.

[ .prettierrc]
<img width="222" alt="image" src="https://github.com/toss/slash/assets/55759551/99663f8e-7a24-47fb-94fd-7064d50a3cd7">

Reply:
Thanks for reviewing. I changed it.